tee-worker/ts-tests/integration-tests/bulk_identity.test.ts

```diff
@@ -3,13 +3,13 @@ import { buildValidations, describeLitentry, buildIdentityTxs, buildIdentityHelp
 import { KeyringPair } from '@polkadot/keyring/types';
 import { ethers } from 'ethers';
 import type { BatchCall } from './common/type-definitions';
-import type { LitentryPrimitivesIdentity } from '@polkadot/types/lookup';
+import type { LitentryPrimitivesIdentity } from 'sidechain-api';
 import type { Call } from '@polkadot/types/interfaces/types';
 import type { Vec } from '@polkadot/types';
 import { assert } from 'chai';
 import { multiAccountTxSender } from './common/transactions';
 import { SubmittableResult } from '@polkadot/api';
-import type { Web3Network } from './parachain-interfaces/identity/types';
+import type { Web3Network } from '../parachain-api/build/interfaces/identity/types';
```

Comment:
Same here; compare the import from `parachain-api` in `common/call.ts` below

Reply:
fixed!